akka-stream/src/main/scala/akka/stream/Materializer.scala

```diff
@@ -36,6 +36,7 @@ abstract class Materializer {
    * stream. The result can be highly implementation specific, ranging from
    * local actor chains to remote-deployed processing networks.
    */
+  // TODO can we deprecate this overload?
```

Comment:
It's a bit weird to have two overloads when this one is equivalent to Attributes.none, and having two methods was the reason for the bug here